syntax/cs.vim

```diff
@@ -44,10 +44,17 @@ syn match	csContextualStatement	/\<partial[[:space:]\n]\+\(class\|struct\|interf
 syn match	csContextualStatement	/\<\(get\|set\)\(;\|[[:space:]\n]*{\)/me=s+3
 syn match	csContextualStatement	/\<where\>[^:]\+:/me=s+5
 
+" Language constructs
+syn keyword	csTypeOf	typeof contained
+syn region	csTypeOfStatement	start="typeof(" end=")" contains=csType, csTypeOf
+syn region	csAttribute	start="^\s*\[" end="\]\s*" contains=csString, csVerbatimString, csCharacter, csNumber, csType, csTypeOfStatement
+
 " Punctuation
 syn match	csBraces	"[{}\[\]]" display
 syn match	csParens	"[()]" display
-syn match	csOpSymbols	"[+\-><=]\{1,2}" display
+syn match	csOpSymbols	"[+\-=]\{1,2}" display
+syn match	csOpSymbols	"[><]\{2}" display
+syn match	csOpSymbols	"\s\zs[><]\ze\s" display
```

Comment:
```suggestion
syn match	csOpSymbols	"\_s\zs[><]\ze\_s" display
```

Using `\_s` instead of `\s` includes newlines in the whitepace. Otherwise an operator before/after a linebreak doesn't match.

Reply:
I can't think of a scenario where you'd have a comparison operator in the first column. Wouldn't it always be inside function scope?

    // highlighted
    if (i
	< LongFunctionName())
    // not highlighted
    if (i<8)
    if (i <
	LongFunctionName())


The trailing \_s does make sense to fix that last case. Will add.